packages/node_modules/@webex/redux-module-conversation/src/reducer.js

```diff
@@ -207,7 +207,7 @@ export default function reducer(state = initialState, action) {
     }
 
     case ACKNOWLEDGE_ADAPTIVE_CARD_SUBMIT_ACTION: {
-      return state.set('lastAcknowledgedcCardActionActivityId', action.payload.activity.id);
+      return state.set('lastAcknowledgedCardActionActivity', action.payload.actionSubmitAcknowledgeObject);
```

Comment:
What are these states used for? I can't find any reference in the codebase for them.

Reply:
I'm using this [here](https://github.com/webex/react-widgets/pull/1054/files#diff-36a8138b14ba83ac03da30246f485bb2R83) for getting the parent and acknowledgmentID of that particular action activity

Reply:
Thanks! I couldn't find it 😃